### DISCUSSION TITLE
Missing some pretrained model weights

### DISCUSSION THREAD

**Discussion by rezafuru** [yoshitomo-matsubara/sc2-benchmark#107] [Q&A] [answered]
Hi, 

I wanted to compare my results using sc2bench but I can't find some important pretrained weights. It seems like the folder `ckpt/ilsvrc2012/input_compression` is missing in the archive you've uploaded (i.e. the weights of the neural compression models). I also cannot find the weights for the pre-trained resnet50 classifier (`ckpt/ilsvrc2012-resnet50.pt`). I could pretrain the models with the configurations youve provided but I'd be very grateful if you could upload them (or tell me whether I've missed them somewhere) as this would save me a lot of time.

Kind regards
A. Furu

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Hi @rezafuru 

For input compression baselines, you won't need to manually download checkpoints, but just use `pretrained: True` in yaml configs so that you can automatically download them via compressai package like [this](https://github.com/yoshitomo-matsubara/sc2-benchmark/blob/main/configs/ilsvrc2012/input_compression/factorized_prior-resnet50.yaml#L46).

The `ckpt:` in our yaml configs for input compression are not used actually unless you have your own trained model weights.
Same for the ResNet-50 model weights, it is from torchvision and you can just use `pretrained: True` like [this](https://github.com/yoshitomo-matsubara/sc2-benchmark/blob/main/configs/ilsvrc2012/input_compression/factorized_prior-resnet50.yaml#L54).

**Reply by rezafuru** [reply]
Hi, thank you very much! Since there was a weights path I assumed that you trained those models separately, and didn't use the pretrained weights from compressai/torchvision.
